scan-command/src/main/resources/scan.properties

```diff
@@ -0,0 +1,74 @@
+#
+# Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+#
+# WSO2 LLC. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+# ------------------------
+# Scan Tool error messages
+# ------------------------
+error.invalid.number.of.arguments=\
+  Invalid number of arguments, expected one argument received {0}
+
+error.empty.package=\
+  ballerina: Package is empty. Please add at least one .bal file.
+
+error.missing.config.field=\
+  Failed to load scan tool configurations: {0} is missing
+
+error.loading.remote.config.file=\
+  Failed to load configuration file with exception: {0}
+
+error.reading.config.file=\
+  Failed to read the configuration file with exception: {0}
+
+error.loading.remote.platform.file=\
+  Failed to retrieve remote platform file with exception: {0}
+
+error.downloading.remote.jar.file=\
+  Failed to download remote JAR file: {0}
+
+error.invalid.json.format=\
+  Invalid JSON format for {0} in plugin '{1}'.\
+  Expected an array, but found:\
+  {2}
+
+error.invalid.json.format.rule=\
+  Invalid JSON format for rule object in plugin '{0}'.\
+  Rule object should contain numeric id, ruleKind and description, but found:\
+  {1}
+
+error.invalid.json.format.rule.kind=\
+  Invalid RuleKind for rule object in plugin '{0}'.\
+  RuleKind should be one of {1}, but found:\
```

Comment:
ruleKind vs RuleKind is not consistent. Please recheck all diagnostics for consistency.